datadog_checks_dev/tests/tooling/test_dep.py

```diff
@@ -63,8 +63,6 @@ def test_package_rich_comparison(package):
     if six.PY3:
         with pytest.raises(TypeError):
             package < 42
-    else:
-        assert (package < 42) is True
```

Comment:
This relies on default implementation of `__cmp__` for the `object` base class, which returns `False` on Windows, let's just get rid of the test